packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+import { IBlockchainProperties, Contracts } from '@energyweb/issuer';
+import { ConfigService } from '@nestjs/config';
+import { DeploymentPropertiesRepository } from './repositories/deploymentProperties/deploymentProperties.repository';
+import { providers, Signer, Wallet } from 'ethers';
+import { getProviderWithFallback } from '@energyweb/utils-general';
+
+@Injectable()
+export class BlockchainPropertiesService {
+    private primaryRPC: string;
+    private issuerPrivateKey: string;
+
+    constructor(
+        private configService: ConfigService,
```

Comment:
you can type `ConfigService<SomeConfigType>` so you can have to type `<string>` when getting specific variables, but that's just a note